frontend/ui/widget/confirmbox.lua

```diff
@@ -79,14 +81,34 @@ function ConfirmBox:init()
     local text_widget = TextBoxWidget:new{
         text = self.text,
         face = self.face,
-        width = math.floor(math.min(Screen:getWidth(), Screen:getHeight()) * 2/3),
+        width = math.floor(math.min(Screen:getWidth(), Screen:getHeight()) * (self.width_percent or 2/3)),
     }
+
+    local image_widget
+    if self.show_icon then
+        --- @todo remove self.image support, only used in filemanagersearch
+        -- this requires self.image's lifecycle to be managed by ImageWidget
+        -- instead of caller, which is easy to introduce bugs
+        if self.image then
```

Comment:
Not sure what you're talking about, I don't see anywhere (neither in this PR, nor in the current codebase, current confirmbox or filemanagerfilesearcher.lua) a use of ConfirmBox having a image= property.

Reply:
Yepp, I have copied this piece from InfoMessage. As I felt the need to 1st disable the icon (and hence an image too) and 2nd I need to make the ConfirmBox wider to fit the screen.

And yes!, the comments need to be adjusted.

But I have no problems to drop this (the ConfirmBox) commit here, and someone (a knowing one) does that. The only thing I want: A wide confirmbox without any noise (image). :)

Reply:
> The only thing I want: A wide confirmbox without any noise (image). :)

We have ButtonDialog that don't have an image - which looks/works just like a ConfirmBox (and can have more buttons). Or ButtonDialogTitle. I thought ButtonDialog would be too large - but it looks like that's what you want :)
So, better if you can use it - than hack ConfirmBox.